runtime/astar/src/lib.rs

```diff
@@ -1591,7 +1591,7 @@ pub type Executive = frame_executive::Executive<
 pub type Migrations = (Unreleased, Permanent);
 
 /// Unreleased migrations. Add new ones here:
-pub type Unreleased = (pallet_dapp_staking::migration::DappStakingCleanupMigration<Runtime>,);
```

Comment:
The cleanup has not been performed yet, we have to keep it. Astar runtime version is still **1301**.